docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>"Adjusted predictions refers to the outcome predicted by a fitted model on a specified scale for a given combination of values of the predictor variables, such as their observed values, their means, or some user specified grid of values" as complex as it reads... it's exactly what it is. So good work here!</p><p> </p><p>Edited to add a question. What do you mean with "link scale"? Is it the scale of the linear predictor, or the scale of mu = g^-1(eta)? I think you mean the scale of "eta" because of the logistic regression example, but I want to confirm.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks, and yeah it is a "mouth full" to say. Yes, I mean the scale of eta because of the logistic regression example. </p><p> </p>